codersdk/organizations.go

```diff
@@ -42,7 +42,8 @@ type OrganizationMember struct {
 
 // CreateTemplateVersionRequest enables callers to create a new Template Version.
 type CreateTemplateVersionRequest struct {
-	Name string `json:"name,omitempty" validate:"omitempty,template_version_name"`
+	Name    string `json:"name,omitempty" validate:"omitempty,template_version_name"`
+	Message string `json:"message,omitempty"`
```

Comment:
As per https://github.com/coder/coder/issues/3775#issuecomment-1608230238, we allow any length of message. We simply warn if a message is over 72 chars.

Reply:
Well, I guess we could use the same limit as the `readme` column, while arbitrary, it at least allows the entire readme to function as a message.

Reply:
I just have a feeling that we should not have an open policy for field limits. I'm with moving this discussion to a wider topic (all fields that are affected).

> Well, I guess we could use the same limit as the readme column, while arbitrary, it at least allows the entire readme to function as a message.

👍

Reply:
@mafredri The limit is more to prevent accidents than abuse IMO.

Reply:
@Emyrk I think preventing both (accidents and abuse) is important. We don't want an angry employee ending their final day with `coder templates push --message "$(dd if=/dev/urandom bs=1M count=1000)"` and causing the deployment to become degraded. (Possibly slow queries, summon OOM killer, UI glitches, etc.)

The above is a contrived example since the shell command limit would prevent it, but doing so via API request would still work.

Reply:
Yes, there is always the rare malicious user. Just saying accidents are more common and enough reason to add a limit.